docs/technical_specifications/PROVIDER.md

```diff
@@ -115,14 +115,11 @@ OIDCFED_PROVIDER_AUTH_CODE_MAX_AGE = getattr(
 )
 ````
 
-- `OIDCFED_PROVIDER_PROFILES_DEFAULT_ACR` default acr value in the Authentication Request if not sumbmitted by RP.
+- `DEFAULT_ACR` default acr value in the Authentication Request if not sumbmitted by RP.
```

Comment:
no please, we prefer OIDCFED_PROVIDER_PROFILES_DEFAULT_ACR with many federations/acrs

this is needed for eidas or any other additional profile in the future